net/proxy_resolution/proxy_config_service_tor.cc

```diff
@@ -15,24 +15,21 @@
 #include "base/values.h"
 #include "base/strings/string_number_conversions.h"
 #include "base/strings/string_util.h"
-#include "content/public/browser/browser_thread.h"
 #include "crypto/random.h"
-#include "net/proxy_resolution/proxy_resolution_service.h"
-#include "net/url_request/url_request_context.h"
+#include "net/proxy_resolution/proxy_config_with_annotation.h"
+#include "net/base/registry_controlled_domains/registry_controlled_domain.h"
 #include "url/third_party/mozilla/url_parse.h"
+#include "url/origin.h"
 
-namespace tor {
+namespace net {
 
-using content::BrowserThread;
 const int kTorPasswordLength = 16;
 // Default tor circuit life time is 10 minutes
-constexpr base::TimeDelta kTenMins = base::TimeDelta::FromMinutes(10);
+constexpr base::TimeDelta kTenMins = base::TimeDelta::FromMinutes(1);
```

Comment:
Doesn't look like ten minutes :)

Reply:
ah, I was doing some testing and forgot to change back

Reply:
fixed